pyproject.toml

```diff
@@ -40,17 +40,14 @@ dependencies = [
     "ipysheet>=0.5.0,<1",
     "plotly>=6.2.0,<7",
     "wop>=2.2.0,<3",
-    "lxml>=4.6.5,<6; python_version<'3.11'",
-    "lxml>=4.9.3,<6; python_version>='3.11'",
+    "lxml>=4.9.3,<7",
```

Comment:
Doesn't this restrict the number of usable version of lxml ? Since we still support 3.10 shouldn't it be kept ? Same reasoning with other dependencies "simplification" you made.